common/OpTestUtil.py

```diff
@@ -1475,12 +1475,12 @@ def get_login(self, host, term_obj, pty, prompt):
         my_user = host.username()
         my_pwd = host.password()
         pty.sendline()
-        rc = pty.expect(['login: ', pexpect.TIMEOUT, pexpect.EOF], timeout=10)
+        rc = pty.expect(['login: ', pexpect.TIMEOUT, pexpect.EOF], timeout=60)
```

Comment:
can we have this as class/function variable? so changing one place is easy in future as  well

Reply:
I see there are too less places the time out change is needed,,, will take up different PR for this